packages/MarkdownEditor-Tests.package/MarkdownInlineTextStylerTest.class/instance/testInterpretTokensWithoutEmphasis.st

```diff
@@ -0,0 +1,6 @@
+tests - interpretation
+testInterpretTokensWithoutEmphasis
+
+	inlineParser parse: 'foo'.
```

Comment:
Shouldn't we only test interpretation of the tokens in here? It conflicts some way with `testParseNormal `.

Reply:
I think the test content is fine but the naming and categories could be improved. The `parse` method does less than before (it stops after interpreting the tokens) and thus what we used to test by first tokenizing and then analyzing those tokens we now test by simply calling parse. We could instead also call `tokenize:` and then `interpretTokens` and then compare with `inlineParser attributes`. Maybe that would show better what we are actually testing.

`testParseNormal` should really be renamed to `testStyleNormal` as we test the `privateStyle` method and no longer `parse`.

Reply:
Please rename the test's instance variables to `inlineTextStyler` or `inlineStyler`.